samples/bluetooth/broadcast_audio_source/overlay-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,23 @@
+# Zephyr Controller tested maximum advertising data that can be set in a single
+# HCI command
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
+
+# Enable support for Broadcast ISO in Zephyr Bluetooth Controller
+CONFIG_BT_CTLR_ADV_ISO=y
+
+# Sufficient ISO PDU length for any BAP LC3 presets (155)
+CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=155
+
+# Number of supported streams
+CONFIG_BT_CTLR_ADV_ISO_STREAM_MAX=2
+CONFIG_BT_CTLR_ISOAL_SOURCES=2
+
+# FIXME: Host needs CONFIG_BT_ISO_TX_MTU + 4 bytes for sequence number, and
+# optionally additional + 4 bytes for timestamp when not using
+# BT_ISO_TIMESTAMP_NONE in bt_iso_chan_send(), otherwise Host tries to fragment
+# ISO data.
+# When Host is fixed, CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE can inherit the
+# CONFIG_BT_ISO_TX_MTU value.
```

Comment:
Do we have an issue for this? Otherwise this FIXME is unlikely to ever get fixed.